<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>龙骨式连续体机器人控制面板</string>
  </property>
  <widget class="QPushButton" name="button_account">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_setting">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_control">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>操纵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_log">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>日志</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="page_stack">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>911</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_account">
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>龙骨式连续体机器人控制面板</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_user">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>用户：</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_password">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>420</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>560</y>
       <width>117</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QComboBox" name="user">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>310</y>
       <width>103</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>User</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
     </item>
    </widget>
    <widget class="QPlainTextEdit" name="password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>请输入密码...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_setting">
    <widget class="QRadioButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>144</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_control">
    <widget class="QRadioButton" name="button_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>144</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>控制</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_log">
    <widget class="QRadioButton" name="button_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>144</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>日志</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
